### PR TITLE
Adding new managed boards and cleaning up naming

### PR DESCRIPTION
**Pull request by DrewJohnsonGT**
Adding more managed boards and updating the mapping structure to make the frontend more dynamic

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,37 +1,37 @@
 import awsAccount from "./boards/aws-accounts/board.json";
+import awsCostAnalysis from "./boards/aws-cost-analysis/board.json";
 import awsIam from "./boards/aws-iam/board.json";
 import awsResources from "./boards/aws-resources/board.json";
 import awsS3Security from "./boards/aws-s3-security/board.json";
-import codeDeps from "./boards/code-deps-licenses/board.json";
+import azureDataStoreSecurity from "./boards/azure-datastore-security/board.json";
```
Comment (DrewJohnsonGT) [inline src/index.ts:6]: Sorting and renaming these imports

File: src/index.ts [hunk 2]
```diff
@@ -73,14 +73,122 @@ export const InsightsDashboards = {
 
 export enum BoardCategory {
   FAVORITE = "FAVORITE",
+  ASSETS_ATTACK_SURFACE = "ASSETS_ATTACK_SURFACE", // "Assets & Attack Surface"
```
Comment (DrewJohnsonGT) [inline src/index.ts:76]: New categories

File: src/index.ts [hunk 3]
```diff
@@ -73,14 +73,122 @@ export const InsightsDashboards = {
...
+  CLOUD_POSTURE = "CLOUD_POSTURE", // "Cloud Posture"
+  DATA_PROTECTION_PRIVACY = "DATA_PROTECTION_PRIVACY", // "Data Protection & Privacy"
   INCIDENT_RESPONSE = "INCIDENT_RESPONSE",
+  SDLC_DEVSECOPS = "SDLC_DEVSECOPS", // "SDLC & DevSecOps"
+  SOFTWARE_SUPPLY_CHAIN = "SOFTWARE_SUPPLY_CHAIN", // "Software Supply Chain Security"
+  USERS_ACCESS = "USERS_ACCESS", // "Users & Access"
   VULNERABILITY_MANAGEMENT = "VULNERABILITY_MANAGEMENT",
 }
 
+export const BoardCategoryTitles = {
```
Comment (DrewJohnsonGT) [inline src/index.ts:86]: image

**Review by ahadrovic — APPROVED** [review]
LGTM

**Pull request merged**